frontend/styles/site.less

```diff
@@ -2225,6 +2226,18 @@ table {
 	}
 }
 
+#chart {
+    width: 1000px;
+    height: 600px;
```

Comment:
Good question.... have to admit that I don't have a good answer yet. I'll create an issue.